app/controllers/helpers.rb

```diff
@@ -0,0 +1,55 @@
+module Helpers
```

Comment:
Beyond being an incorrect pattern, this `Helpers` module is responsible for too much. It knows about logging people in and checking if people can edit tweets. These functionalities should be in different concerns or modules and correctly mixed into `ApplicationHelper` with the `helpers` macro described above.